src/main/kotlin/minesweeper/domain/Location.kt

```diff
@@ -0,0 +1,20 @@
+package minesweeper.domain
+
+class Location private constructor(private val y: Int, private val x: Int) {
+
+    fun getConvertIndex(width: Int): Int {
+        return y * width + x
+    }
+
+    companion object {
+        fun of(location: String?): Location {
+            require(location != null && location.isNotBlank()) { "빈 값을 입력하면 안됩니다." }
```

Comment:
`String?`에 대해 호출할 수 있는 `isNullOrEmpty()` 또는 `isNullOrBlank()`가 있습니다.

```suggestion
            require(!location.isNullOrBlank()) { "빈 값을 입력하면 안됩니다." }
```